webclient/src/components/Home/Description.tsx

```diff
@@ -0,0 +1,198 @@
+import { MdOutlineMap } from 'react-icons/md';
+import { Link } from 'react-router-dom';
+import { Avatar, Box, Flex, Heading, ListItem, Stack, Text, UnorderedList } from '@chakra-ui/react';
+import { ArrowForwardIcon, Icon, InfoOutlineIcon } from '@chakra-ui/icons';
+import { OstNavLink } from '../Elements/OstLink';
+import { OstButton } from '../Elements/OstButton';
+
+type StepCardProps = {
+  children: React.ReactNode;
+  title: React.ReactNode;
+  subTitle: string;
+};
+
+const StepCard = ({ children, title, subTitle }: StepCardProps) => {
+  return (
+    <Stack
+      alignItems="left"
+      borderRadius={8}
+      textStyle="navigationLarge"
+      background="white"
+      // 各要素の幅を均等にするために指定
+      width="100%"
+    >
+      <Stack
+        background="#42667A"
+        color="white"
+        px={6}
+        py={4}
+        // HACK: 上側だけ角を丸くする
+        borderRadius="8px 8px 0 0"
```

Comment:
```suggestion
        borderTopRadius={8}
```
こう書くと、border-top-left-radius + border-top-right-radius になったりします。

Reply:
片側だけ丸くできるpropsが用意されているのですね！他にも似たような指定をした箇所があったので、まとめて修正しました。

845c0d4c3eac9fb1aab8986e76121116d8b9032d